### PR TITLE
Add job to build Arch Linux and Debian packages

### PR DESCRIPTION
**Pull request by ljmf00**
Signed-off-by: Luís Ferreira <contact@lsferreira.net>

### CODE REVIEW

**Comment by ljmf00** [comment]
Tested on my local machine

**Comment by iK4tsu** [comment]
LGTM

**Review by iK4tsu — APPROVED** [review]

**Review by iK4tsu — APPROVED** [review]

**Comment by iK4tsu** [comment]
Why is it in WIP though?

**Comment by ljmf00** [comment]
> Why is it in WIP though?

I planned to merge it with locked versions instead of rely on the master. That would be cool for future maintenance.

**Comment by ljmf00** [comment]
@iK4tsu can you take a look now?

File: .github/workflows/packaging.yml [hunk 1]
```diff
@@ -1,17 +1,91 @@
...
+      - name: Clone autopsy-bin build repo from AUR
+        run: |
+          git clone https://aur.archlinux.org/autopsy-bin.git $HOME/autopsy-bin
+          (cd $HOME/autopsy-bin; git checkout $PACKAGE_AUTOPSY_SHA)
+      - name: Build autopsy-bin
+        run: |
+          chown user -R $HOME/autopsy-bin
+          cd $HOME/autopsy-bin
+          extra-x86_64-build -- -U user -I "$(find "/tmp/artifacts" -name 'sleuthkit-java-*.pkg.tar.zst')"
+          cp *.pkg.* /tmp/artifacts
```
Comment (iK4tsu) [inline .github/workflows/packaging.yml:47]: Why not `mv`?

File: .github/workflows/packaging.yml [hunk 2]
```diff
@@ -1,17 +1,91 @@
...
+            --iteration "$PACKAGE_SLEUTHKIT_JAVA_REL" \
+            --no-auto-depends \
+            -d "openjdk-8-jre" \
+            -d "sleuthkit-java (>= ${PACKAGE_SLEUTHKIT_JAVA_VERSION})" \
+            -d "testdisk" \
+            -d "openjfx" \
+            -a amd64 \
+            -s pacman autopsy-bin-${PACKAGE_AUTOPSY_VERSION}-${PACKAGE_AUTOPSY_REL}-x86_64.pkg.tar.zst
+          mkdir -p /tmp/artifacts
+          cp *.deb /tmp/artifacts
```
Comment (iK4tsu) [inline .github/workflows/packaging.yml:87]: ditto

**Review by iK4tsu — CHANGES_REQUESTED** [review]

File: .github/workflows/packaging.yml (hunk 1 above)
Comment (ljmf00) [reply .github/workflows/packaging.yml:47]: Initially, I had `mv` but changed `cp` for further debug. But it's really pointless, although I'll change it. Makes more sense

Comment (ljmf00) [reply .github/workflows/packaging.yml]: Done

File: .github/workflows/packaging.yml (hunk 2 above)
Comment (ljmf00) [reply .github/workflows/packaging.yml]: Done

**Review by iK4tsu — APPROVED** [review]

**Comment by iK4tsu** [comment]
LGTM

**Pull request merged**